### PR TITLE
Add release note for release 1.0

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: CHANGELOG_1.0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## KubeEdge v1.3 Release Notes
```
Comment (k82cn) [inline CHANGELOG_1.0.md:1]: KubeEdge?

File: CHANGELOG_1.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### 1.3 What's New
```
Comment (k82cn) [inline CHANGELOG_1.0.md:3]: s/1.3/1.0

File: CHANGELOG_1.0.md (hunk 1 above)
Comment (hzxuzhonghu) [reply CHANGELOG_1.0.md:1]: I forgot to force push

File: CHANGELOG_1.0.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- Add mindspore example (#845, @lyd911)
+- Add golangCi-lint check (#799, #821, #824, #825, #827, #829, #830, #833, #835, #841. @masihtehrani @daixiang0 @Thor-wl)
+- Allow specifying admission webhook port (#832, @hzxuzhonghu)
+- Add support for most requested priority (#831, @daixiang0)
+- Set pod DNSPolicy to ClusterFirstWithHostNet when hostnetwork set (#779, @hzxuzhonghu)
+- Add bestNodeFn for plugins to select best node of its own (#790, @jiangkaihua)
+- Add e2e cases for reclaim (#898, @alcorj-mizar)
+- Add e2e cases for preempt (#892, @Thor-wl)
+- Add e2e for queue (#872, @Thor-wl)
+- Add test code for queue controller (#858, @alcorj-mizar)
```
Comment (hzxuzhonghu) [inline CHANGELOG_1.0.md:38]: /hold

need to add link

**Pull request merged**